tests/Aspire.Hosting.Testing.Tests/TestingBuilderTests.cs

```diff
@@ -230,7 +230,7 @@ public async Task CrashTests(bool genericEntryPoint, string crashArg)
             var exception = await Assert.ThrowsAsync<InvalidOperationException>(() =>
             genericEntryPoint ? DistributedApplicationTestingBuilder.CreateAsync<Projects.TestingAppHost1_AppHost>([$"--crash-{crashArg}"], cts.Token).WaitAsync(cts.Token)
                                 : DistributedApplicationTestingBuilder.CreateAsync(typeof(Projects.TestingAppHost1_AppHost), [$"--crash-{crashArg}"], cts.Token).WaitAsync(cts.Token))
-                .ConfigureAwait(false);
+;
```

Comment:
NIT: Fix formatting by removing new-line.

Reply:
I'll do a cleanup pass once all the tests are passing.